lib/galaxy/schema/schema.py

```diff
@@ -690,13 +683,96 @@ class HDCASummary(HistoryItemCommon):
         description="The type of job (model class) that produced this dataset collection. Used to track the state of the job.",
     )
     contents_url: RelativeUrl = ContentsUrlField
+    collection_id: Optional[EncodedDatabaseIdField] = Field(
```

Comment:
If you want it in one response model, but not another (I assume that's why you made it optional) you should place it in the response model that returns the collection_id, not the common model.